contracts/vendor/starkware-libs/starkgate-contracts/contracts/starkware/starknet/solidity/IStarknetMessaging.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0.
+pragma solidity ^0.6.12;
+
+import "./IStarknetMessagingEvents.sol";
+
+interface IStarknetMessaging is IStarknetMessagingEvents {
+    /**
+      Sends a message to an L2 contract.
+
```

Comment:
remove extra line.  Also why do we need to implement this ourselves?  Does Starknet not provide this interface?

Reply:
It does, but they don't distribute it properly so we include it in our project as vendor contract. We also update the pragma to include v0.8 for this interfaces we need to use.